README.md

```diff
@@ -1,2 +1,58 @@
-# talenta-api
-API HR Talenta for ClockIn and ClockOut
+# Talenta-api
+API HR Talenta for ClockIn and ClockOut, so you don't need to open the app and clockin/clockout
+
+# Requirement
+1. npm
+2. nodejs
+3. yarn
+
+# Installation
+Just run `yarn install`
+
+# How to Run Program
+
+At first you need to specify this three variable `Cookie`, `Latitude` and `Longitude`
+
+## How to get Cookie?
+
+1. you need to login with this url `https://account.mekari.com/users/sign_in?app_referer=Talenta` and sign in
+![](img/login.png)
+2. after login on the browser preff `F12`
+3. the new window/new window on bottom screen will popup and switch to `Application` tab, pick `Cookie`->`https://hr.talenta.co` and then scroll down until get the Cookie with name `PHPSESSID` or `_identity` 
```

Comment:
I think we need only _identity

Reply:
I give the option to use the `PHPSESSID` or `_identity` so the user understand that we can use one of it, or what do you think ? just `_identity` is enough?

Reply:
hmm, I only use _identity tho, have you tested it for PHPSESSID?
If you have tested it, I think its okay